### ISSUE TITLE
QUESTION: How do I edit data in a row or delete a row?

### ISSUE THREAD

**Issue by RocksteadyTC** [beekeeper-studio/beekeeper-studio#743]
**Details:**
I am trying to learn how to delete a row of a table or edit data in a field of a row in a table, and looking at the doc it says to right click the row, and in a merged feature it says you can double click the field to edit, but none of those works for me, single click and double click as well as shift, alt and ctrl click all just select the text in the field.  delete key does not delete row or field data.

**Version Information [might be relevant to your issue]**
Operating System: manjaro linux (latest)

The DB Im connecting to is a remote Postgres cluster (the hosted digitalocean db cluster solution)

App Version [help -> about]: 1.12.0 (latest in the repos) as well as the latest build pushed 5 hours ago in Releases on Github

**Screenshot / GIF [if appropriate]**:
A screenshot of the app if that helps
![image](https://user-images.githubusercontent.com/34389545/127075336-e91dc5a1-6e69-4085-bc71-73d81f9230ea.png)

Thanks to you guys for making an awesome app! Im curious if there is something I'm overlooking, so I'm going to keep poking at this until I hear back from you guys.

**Comment by ghstng** [comment]
Right click a table in the sidebar and click 'View Structure" or click 'Structure' in the bottom left of the statusbar
![snip1](https://user-images.githubusercontent.com/5122567/127077825-674641f2-d2d2-463c-8f8d-dd9e7ac2c4d8.png)
![snip2](https://user-images.githubusercontent.com/5122567/127077828-1645989a-8d15-4470-9bd4-1968a1f9a5bb.png)

**Comment by RocksteadyTC** [comment]
Possibly the problem is that this table does not have a primary key.

**Comment by RocksteadyTC** [comment]
Yes, that did it. As soon as I added a column as primary key, the issue resolved itself. 

It's worth mentioning, I only noticed this by switching to another GUI client and attempting the same operation, which yielded a warning that since this table had no primary key, it would be read only. This warning is not present in beekeeper, that I've noticed. Perhaps some type of warning like this might help others in my situation.

Regardless, great piece of software you have here. I look forward to what you guys decide on with regard to ways we can help fund development of this great product.

Take care, have a good week. I'd normally close this issue, but IDK if it will end up as a feature request or what, so I'll leave that up to you @rathboma
